AudioKit/Core Classes/AKManager.h

```diff
@@ -29,6 +29,9 @@ NS_ASSUME_NONNULL_BEGIN
 /// Determines whether or not to log
 @property BOOL isLogging;
 
+/// Place to put lines of testing logs
+@property NSMutableArray *testLog;
+
```

Comment:
This is only added to in tests.  Not only for TravisCI, but locally.  And also, I will be replacing this soon.